### PR TITLE
Release 1.7.6.3

### PR DESCRIPTION
**Pull request by Progi1984**

### CODE REVIEW

File: images/1.7.6.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM prestashop/base:5.6-apache
+LABEL maintainer="Thomas Nabord <thomas.nabord@prestashop.com>"
+
+ENV PS_VERSION 1.7.6.3
+
+# Get PrestaShop
+ADD https://www.prestashop.com/download/old/prestashop_1.7.6.3.zip /tmp/prestashop.zip
```
Comment (matks) [inline images/1.7.6.3/Dockerfile]: Woah 😮 I have no idea what is this folder https://www.prestashop.com/download/old/

File: images/1.7.6.3/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM prestashop/base:5.6-apache
+LABEL maintainer="Thomas Nabord <thomas.nabord@prestashop.com>"
```
Comment (matks) [inline images/1.7.6.3/Dockerfile]: @Quetzacoalt91 Are you still OK to be labeled the maintainer of these images ? Maybe one day we should put a more generic email address, such as core@prestashop.com or docker-core@prestashop.com ?

Comment (Quetzacoalt91) [reply images/1.7.6.3/Dockerfile]: Yes we should have a generic address, probably `docker@prestashop` as several teams could manage this images. WDYT?

File: images/1.7.6.3/Dockerfile (hunk 1 above)
Comment (Progi1984) [reply images/1.7.6.3/Dockerfile]: it was automated by `generateAll.sh`

File: images/1.7.6.3/Dockerfile (hunk 2 above)
Comment (matks) [reply images/1.7.6.3/Dockerfile]: See https://github.com/PrestaShop/docker/pull/202 (email address was chosen by Pablo)

**Pull request merged**